include/liblwm2m.h

```diff
@@ -97,6 +97,8 @@ extern "C" {
 #error "LWM2M_BOOTSTRAP and LWM2M_BOOTSTRAP_SERVER_MODE cannot be defined at the same time!"
 #endif
 
+extern uint16_t coap_block_size;
```

Comment:
I don't think there are any other variables publicly exported by Wakaama. I would suggest moving this to core/internals.h and creating a function to set it. That would also give a point to notify a CoAP stack (if we support a different one in the future) of a change in the block size.

On another note, anything meant to be accessed by the user's code should start with lwm2m.